### PR TITLE
Adds a contribution guide

### PR DESCRIPTION
**Pull request by HammadB**
With much inspiration from https://github.com/chroma-core/chroma/issues/968!

### CODE REVIEW

**Comment by HammadB** [comment]
@mcgeestocks Curious what you think!

File: docs/contributing.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+
+A CIP starts in the "Proposed" state, then moves to "Under Review" once the Chroma team has reviewed it and is considering it for implementation. Once the CIP is approved it will move to the "Accepted" state and the implementation can begin. Once the implementation is complete the CIP will move to the "Implemented" state. If the CIP is not approved it will move to the "Rejected" state. If the CIP is withdrawn by the author it will move to the "Withdrawn" state.
```
Comment (jeffchuber) [inline docs/contributing.md:38]: should we link to an example?

File: docs/contributing.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+# 🍻 Contributing
+
+We welcome all contributions, bug reports, bug fixes, documentation improvements, enhancements, and ideas.
+
+## Getting Started
+The Chroma codebase is hosted on [Github](https://github.com/chroma-core/chroma) and issues are tracked on [Github Issues](https://github.com/chroma-core/chroma/issues). Please report any issues you find there making sure to fill out the correct [form for the type of issue you are reporting](https://github.com/chroma-core/chroma/issues/new/choose). In order to run Chroma locally you can follow the [Development Instructions](https://github.com/chroma-core/chroma/blob/main/DEVELOP.md).
+
+If you want to contribute and aren't sure where to get started you can search for issues with the [Good first issue](https://github.com/chroma-core/chroma/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22) tag.
+
+The Chroma documentation (including this page!) is hosted on [Github](https://github.com/chroma-core/docs) as well. If you find any issues with the documentation please report them on the Github Issues page for the documentation [here](https://github.com/chroma-core/docs/issues).
```
Comment (jeffchuber) [inline docs/contributing.md:15]: the roadmap section also currently has a discussion of great areas for contribution

File: docs/contributing.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+sidebar_position: 10
+title: "🍻 Contributing"
+---
+
+# 🍻 Contributing
+
+We welcome all contributions, bug reports, bug fixes, documentation improvements, enhancements, and ideas.
+
+## Getting Started
+The Chroma codebase is hosted on [Github](https://github.com/chroma-core/chroma) and issues are tracked on [Github Issues](https://github.com/chroma-core/chroma/issues). Please report any issues you find there making sure to fill out the correct [form for the type of issue you are reporting](https://github.com/chroma-core/chroma/issues/new/choose). In order to run Chroma locally you can follow the [Development Instructions](https://github.com/chroma-core/chroma/blob/main/DEVELOP.md).
```
Comment (jeffchuber) [inline docs/contributing.md:11]: maybe this would be better as bullets? more digestable and each sentence is pretty standalone

**Review by jeffchuber — APPROVED** [review]
added some nits mostly in the spirit of always getting better. but good as is

File: docs/contributing.md (hunk 3 above)
Comment (HammadB) [reply docs/contributing.md:11]: copy

File: docs/contributing.md (hunk 1 above)
Comment (HammadB) [reply docs/contributing.md:38]: Once we have a good, merged example, we can link to it. Right now we don't unfortunately

Comment (HammadB) [reply docs/contributing.md:38]: We could link to CIP-1 and merge it as under review.

**Review by mcgeestocks — COMMENTED** [review]
Looks great @HammadB, also agree w. @jeffchuber on the bullet point comment.

File: docs/contributing.md (hunk 1 above)
Comment (HammadB) [reply docs/contributing.md:38]: Will address this once i merge CIP-1 - need to make sure liquan is ok with merging and he's OOO

**Pull request merged**